examples/demo.js

```diff
@@ -8,7 +8,27 @@ var defaultHtmlSequencerUi = require('./lib/defaultHtmlSequencerUi.js'),
 window.onload = function () {
   sequencer = ImageSequencer();
 
-  function refreshOptions() {
+  options = {
+      sortField: 'text',
+      openOnFocus: false,
+      onInitialize: function () {
+          var that = this;
```

Comment:
Is there a reason why you created this variable?

Reply:
if you mean options variable - because of codeclimat tests.
It added a default value for it so this code should not break anything

Reply:
no, the `that` variable? It seems redundant to me.

Reply:
because we cant use the parent’s “this” in
```
this.$control.on("click", function () {
            that.ignoreFocusOpen = true;
            setTimeout(function () {
                that.ignoreFocusOpen = false;
            }, 50);
        });
```

Reply:
Can you instead use an arrow function? Arrow functions do not have their own scoped `this`.

Reply:
I’m not sure that old browsers support arrow functions

Reply:
We already use arrow functions. Don't worry. Old browsers don't support many things. Ultimately, the benefits of ES6 are much more. Just because of browser support, we can't ignore the benefits. Anyway, I don't think anyone will use IS on IE. Similarly, we dropped support for NodeJS 6.0 because it doesn't support GPU acceleration which is crucial. Thanks.

Reply:
Like this?
```
this.$control.on("click", (e) => {
    this.ignoreFocusOpen = true;
    setTimeout(function () {
        this.ignoreFocusOpen = false;
    }, 50);
});
```

Reply:
yes. great.

Reply:
So can you accept gci task now?